Services/Sync/SyncModel/BasicModelInterface.php

```diff
@@ -3,9 +3,9 @@
 
 interface BasicModelInterface
 {
-    function getItem($normalizedData);
-    function restoreItem($normalizedData);
-    function createItem($normalizedData);
-    function updateItem($item, $normalizedData);
-    function getId($item);
+    function getItem($normalizedData, $parentItem = null);
```

Comment:
gotta explain what the parentItem thing is exactly, i am interested

Reply:
@qrakhen it's possible to define "additionalSyncModels". this enables syncing data in relational-models, therefore the need to know a parentItem. I'm still doubtfull it's a good idea to mix syncModel and additionalSyncModel them into one interface... but two different interfaces would also require to use a different function-name to enable the use of the same sync-model-class in both contexts, implementing both interfaces.